deltachat-ios/en.lproj/Localizable.strings

```diff
@@ -615,10 +615,4 @@
 "a11y_delivery_status_invalid" = "Invalid delivery status";
 "a11y_remove_account_btn_label" = "Remove account";
 "a11y_message_context_menu_btn_label" = "Message actions";
-"a11y_background_preview_label" = "Background preview";
-
-// Translators: These strings are only used on iOS
```

Comment:
Why did you remove the follwoing lines? 
import_contacts and import_contact_message are still in use.

`no_account_setup` is unused though.

Reply:
import_contacts and import_contact_message cannot be deleted. They are referenced in the project. 
The same keys exist in untranslated.xml but that is only the source for pulling strings into this file. untranslated.xml is not referenced in the project!

Reply:
I understand what you mean. However, when we import the entries in the untranslated.xml file into our Android project, the strings I have already deleted will come back. Please merge this.